src/configureWorkspace/configurePython.ts

```diff
@@ -3,68 +3,224 @@
+    let command = "";
+
+    if (projectType === "general") {
+        if ((target as PythonFileTarget).file) {
+            command = `CMD ["python", "${(target as PythonFileTarget).file}"]`;
+        } else {
+            command = `CMD ["python", "-m", "${(target as PythonModuleTarget).module}"]`;
+        }
+    } else if (projectType === "django") {
+        command = `CMD ["gunicorn", "--bind", "0.0.0.0:${ports ? ports[0] : PythonDefaultPorts[projectType]}", "${serviceName}.wsgi"]`;
+    } else if (projectType === "flask") {
+        command = `CMD ["gunicorn", "--bind", "0.0.0.0:${ports ? ports[0] : PythonDefaultPorts[projectType]}", "${serviceName}:app"]`;
+    } else {
+        // Unlikely
+        throw new Error(`Unknown project type: ${projectType}`);
+    }
+
+    return pythonDockerfile
+        .replace(/\$expose_statements\$/g, exposeStatements)
+        .replace(/\$cmd\$/g, command);
+}
+
+function genDockerCompose(serviceName: string, ports: number[]): string {
+    return dockerComposefile
+        .replace(/\$service_name\$/g, serviceName)
+        .replace(/\$ports\$/g, getComposePorts(ports));
+}
+
+function genDockerComposeDebug(serviceName: string, projectType: PythonProjectType, ports: number[], target: PythonFileTarget | PythonModuleTarget): string {
+    const defaultDebugOptions : PythonExtensionHelper.DebugLaunchOptions = {
+        host: "0.0.0.0",
+        port: PythonDefaultDebugPort,
+        wait: true
+    };
+
+    const args = inferPythonArgs(projectType, ports);
+    const launcherCommand = PythonExtensionHelper.getRemotePtvsdCommand(target, args, defaultDebugOptions);
+    const entrypoint = "python ".concat(launcherCommand);
+
+    return dockerComposeDebugfile
+        .replace(/\$service_name\$/g, serviceName)
+        .replace(/\$dbg_volume\$/g, `- ${PythonExtensionHelper.getLauncherFolderPath()}:/pydbg`)
+        .replace(/\$entrypoint\$/g, entrypoint)
+        .replace(/\$ports\$/g, getComposePorts(ports, PythonDefaultDebugPort));
+}
+
+function genRequirementsFile(projectType: PythonProjectType): string {
+    let contents = '# Add requirements when needed'
+
+    switch (projectType) {
+        case "django":
+            contents = djangoRequirements;
+            break;
+        case "flask":
+            contents = flaskRequirements
+            break;
+        default:
+    }
+
+    return contents;
+}
+
+async function initializeForDebugging(context: ScaffolderContext, dockerfile: string, ports: number[], generateComposeFiles: boolean,
+                                      target: PythonFileTarget | PythonModuleTarget, projectType: PythonProjectType): Promise<void> {
+    const scaffoldContext: DockerDebugScaffoldContext = {
+        folder: context.folder,
+        platform: "python",
+        actionContext: context,
+        dockerfile: dockerfile,
+        generateComposeTask: generateComposeFiles,
+        ports: ports
+    }
+
+    const pyOptions: PythonScaffoldingOptions = {
```

Comment:
Nit:

```js
const pyOptions: PythonScaffoldingOptions = {
    projectType,
    target
};
```

Reply:
I don't have much preference to break the tie, so which style is more consistent with the repo?

Reply:
Alright, I looked in a random set of files in the repo and it seems the longer form is more dominant. Sorry, Phil :).

Reply:
`string.Method` or `String.Method`? 😄

Reply:
I don't know if I'd look in this repo for any sort of consistent guidance on style.  It's gone through at least three significant changes in ownership, each with their own particular style. (I'm going to use the short form whenever/wherever I can.) :-)